package.json

```diff
@@ -101,7 +101,7 @@
 
     "protractor": "^3.2.2",
 
-    "parse5": "^1.5.1",
+    "parse5": "^2.1.5",
```

Comment:
Then let's remove it^^ Unfortunately I destroy the fork after every PR you merge and fork again for every PR.